terraform/cyhy_nmap_ec2.tf

```diff
@@ -22,7 +22,7 @@ data "aws_ami" "nmap" {
 
 resource "aws_instance" "cyhy_nmap" {
   ami = "${data.aws_ami.nmap.id}"
-  instance_type = "${local.production_workspace ? "r5.24xlarge" : "t3.micro"}"
+  instance_type = "${local.production_workspace ? "t3.micro" : "t3.micro"}"
```

Comment:
This line and the `volume_size` line below it could obviously be simplified if the values don't depend on whether or not we're in a production workspace.  What is the intent?  Are you going to see if this works out and follow up?

Reply:
Yes, I left it like that in case we end up switching to different instance types in the future.  Plus, it's consistent with the code over in the vulnscan instance.  If it looks like we are never going to have different instance types between Production and non-Prod, then I'm all for making the change to simplify this line.